chart/jetstack-secure-gcm/templates/billing-agent-config.yml

```diff
@@ -6,8 +6,9 @@ metadata:
   labels:
     helm.sh/chart: "{{ .Chart.Name  }}"
     app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
-    app.kubernetes.io/component: ubbagent
+    app.kubernetes.io/name: "{{ .Chart.Name | trunc 63 | trimSuffix "-" }}"
     app.kubernetes.io/instance: "{{ .Release.Name | trunc 63 | trimSuffix "-" }}"
+    app.kubernetes.io/component: ubbagent
```

Comment:
Can I wondered if we could use more descriptive values for these component names, e.g. `Google Cloud Marketplace Billing Agent`....

BUT WAIT why is this still here as a separate deployment, if you have now deployed the ubbagent as a cert-manager side-car?

Reply:
...ah, this isn't the deployment, but the config for ubbagent...ignore me. 
I see the deployment is deleted below.

Reply:
Apologies for the messy PR :-(